spec/requests/sellers_spec.rb

```diff
@@ -126,6 +126,11 @@
         expected_json['target_amount'] = 1_000_000
         expected_json['amount_raised'] = 0
         expected_json['locations'] = []
+        expected_json['donation_amount'] = 0
+        expected_json['gift_card_amount'] = 0
+        expected_json['num_contributions'] = 0
+        expected_json['num_donations'] = 0
+        expected_json['num_gift_cards'] = 0
```

Comment:
Seems like a lot of copying and pasting but we're asserting on the same values. Maybe we could refactor this out into a big expect and just call the method?